ocs_ci/deployment/helpers/tnf_helpers.py

```
@@ -0,0 +1,772 @@
+    Args:
+        node_name (str): Node name
+        device_path (str): Device path (e.g. /dev/vdb)
+
+    Returns:
+        str: The /dev/disk/by-id/ path, or the original path if resolution fails
+    """
+    _fix_rotational_flag_if_virtual(node_name, device_path)
+    _ensure_disk_by_id_symlink(node_name, device_path)
+
+    try:
+        result = exec_cmd(
```

Ruff (0.16.1)

[error] Function call with `shell=True` parameter identified, security issue

(S604)